response_operations_ui/templates/messages.html

```diff
@@ -19,7 +19,11 @@
                 })
             %}
                 {% for message in messages %}
-                    <p id="flashed-message-{{ loop.index }}">{{ message }}</p>
+                    <p id="flashed-message-{{ loop.index }}">{{ message }}
+                        {% if thread_url %}
```

Comment:
If we ever need to extend this pattern to be used with other pages we might want it with a more extensible name like `flashed_url` or something. But this is also fine as this instance is specific to the messages and we have another flashing html that is used for everything else